interfaces/Portalicious/src/app/components/form/form-default.component.html

```diff
@@ -9,11 +9,12 @@
       [error]="mutation().isError() && mutation().failureReason()?.message"
       class="mt-8"
     />
-    <p-button
-      [label]="submitButtonText()"
-      type="submit"
-      rounded
-      [loading]="mutation().isPending()"
-    />
   </div>
+  <p-button
```

Comment:
I don't think this is a logical solution... 😕 🤔 

I think some magical flex-box alignment classes need to be juggled here..